samples/bot-commands-menu/nodejs/bots/teamsConversationBot.js

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const {
+    CardFactory,
+    TeamsActivityHandler,
+    TurnContext
+} = require('botbuilder');
+
+const FlightsDetailsCardTemplate  = require('../resources/flightsDetails.json');
+
+const SearchHotelsCardTemplate  = require('../resources/searchHotels.json');
+
+class TeamsCommandsMenuBot extends TeamsActivityHandler {
+    constructor() {
+        super();
+
+        this.onMessage(async (context, next) => {
+            
+        var activity = TurnContext.removeMentionText(context.activity);
+
+        if (activity != null)
+        {
+            if (context.activity.text != null) 
+            {
+                const text = context.activity.text.trim().toLocaleLowerCase();
+                if (text.includes('search flights')) 
+                {
+                    await this.SearchFlightsReaderCardAsync(context);
+                } 
+                else if (text.includes('search hotels')) 
+                {
+                    await this.SearchHotelsReaderCardAsync(context);
+                } 
+                else if (text.includes('help')) 
+                {
+                    await context.sendActivity('Displays this help message.');
+                } 
+                else if (text.includes('best time to fly')) 
+                {
+                    await context.sendActivity('Best time to fly to London for a 5 day trip this summer.');
```

Comment:
Do you mean to say "Best time to fly to London for a 5 day trip **is** summer."?

Reply:
Updated below content.

Best time to fly to London for a 5 day trip is summer.